app/controllers/friendships_controller.rb

```diff
@@ -0,0 +1,34 @@
+class FriendshipsController < ApplicationController
+
+  def new
+    @friendships = Friendship.new
+  end
+
+  def create
+    user = User.find(params[:id])
+    @friendship = Friendship.create(user_id: current_user.id, friend_id: user.id, confirmed: false)
+    if @friendship.save
+      flash[:notice] = "Friend request sent successfully!"
+      redirect_to users_path
+    else
+      flash[:alert] = "Cannot send friend request"
+      redirect_to users_path
```

Comment:
Use `render:` instead of `redirect` in the negative scenarios to avoid reloading the page.